gdk4-wayland/Cargo.toml

```diff
@@ -24,6 +25,8 @@ gio = {git = "https://github.com/gtk-rs/gtk-rs-core", features = ["v2_66"]}
 glib = {git = "https://github.com/gtk-rs/gtk-rs-core", features = ["v2_66"]}
 libc = "0.2"
 wayland-client = {version = "0.28", features = ["use_system_lib"]}
+khronos-egl = {version = "4.1.0", optional = true}
+xkb = {version = "0.2.1", optional = true}
```

Comment:
The new dependencies's docs links needed to be added at https://github.com/gtk-rs/gtk4-rs/blob/master/.github/workflows/docs.yml#L38 for the docs to be properly linking to those crates

Reply:
done

Reply:
(I don't know how to verify it yet)

Reply:
Let's see how that goes once merged :)

Reply:
resolving, assuming it works for now